### ISSUE TITLE
Auto map rotation

### ISSUE THREAD

**Issue by DryByte** [SpadesX/SpadesX#51]
**Is your feature request related to a problem? Please describe.**
This probably is related to #47 

**Describe the solution you'd like**
Adding a maps folder, then add an option to config for "auto_rotation", that will read all maps into maps folder then adding it to rotation, overwriting the currently rotation setted on config.

**Describe alternatives you've considered**
Using the #47 combined with the above idea.

**Comment by Haxk20** [comment]
Yep something similar is planned later on :) I will keep this opened so we can keep track of it

**Comment by lancilloty** [comment]
@Haxk20   Enhancement label?

**Comment by Fran6nd** [comment]
@Haxk20 this one should be closed now :)
